PennyMe/MachineChangedRequest.swift

```diff
@@ -0,0 +1,326 @@
+//
+//  MachineChangedRequest.swift
+//  PennyMe
+//
+//  Created by Nina Wiedemann on 04.11.23.
+//  Copyright © 2023 Jannis Born. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import MapKit
+import Combine
+
+import SwiftUI
+import MapKit
+import CoreLocation
+
+// variable defining how large the shown region is when changing coordinates
+let regionInMeters: Double = 100
```

Comment:
This is just the size of the initial region shown on the map. The user can zoom in and move the map. I thought a 100 x 100 window is suitable for a start because most of the time, it should just be about moving the pin to the right spot in a building or so.